automation/vars/default.yaml

```diff
@@ -31,8 +31,10 @@ vas:
             osdpd edpm-deployment-pre-ceph --for condition=Ready
             --timeout=1500s
         values:
-          - name: edpm-values
-            src_file: values.yaml
+          - name: edpm-nodeset-values
```

Comment:
hmm if we keep it like that, it's still one stage. I expected to get another block to ensure the nodeset is on its own, something like:
```YAML
- path: examples/va/hci/edpm-pre-ceph/nodeset
  wait_conditions:
    - oc -n openstack wait <nodeset is ready/up-to-date/whatever>
  values:
    - name: edpm-nodeset-values
      src_file: values.yaml
  build_output: nodeset.yaml
- path: examples/va/hci/edpm-pre-ceph/deployment
  wait_conditions:
    - oc -n openstack wait <dataplane is ready/up-to-date/whatever>
  values:
    - name: edpm-deployment-values
      src_file: values.yaml
  build_output: dataplane.yaml
```
or something like that... ?